keel-core/src/test/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunnerTests.kt

```diff
@@ -67,7 +66,7 @@ internal class VerificationRunnerTests {
         name = "fnord-manifest",
         serviceAccount = "jamm@illuminati.org",
         artifacts = setOf(
-          DockerArtifact(name = "fnord", reference = "fnord-docker", branch = "main")
+          DockerArtifact(name = "fnord", reference = "fnord-docker")
```

Comment:
The removal of the `branch` argument in this file looks like an accidental revert of unrelated code.